classes/output/login.php

```diff
@@ -23,23 +23,27 @@
 
 namespace local_login\output;
 
+use cache;
+use Mustache_Engine;
+use Mustache_Exception;
+
 /**
  * Output buttons.
  */
 class login {
 
     /**
-     * Output login buttons to page.
+     * Output login page.
      *
      * @param string $wantsurl
      * @return string
      */
-    public static function output_login_options($wantsurl) {
-        // Could have used a renderer, but just nicer to have everything in the same file.
+    public static function render_login($wantsurl) {
+        global $CFG;
         $config = get_config('local_login');
-        // Output.
-        $output = '';
-        $output .= \html_writer::start_tag('div', array('id' => 'local-login-options', 'class' => 'container-fluid'));
+        $context = new \stdClass();
+
+        $idplist = [];
         $count = 0;
```

Comment:
Why do we need a count variable? Why not just use `array_count($idplist)` at the end?